pyQuARC/__init__.py

```diff
@@ -15,5 +17,24 @@
 with open(f"{ABS_PATH}/version.txt") as version_file:
     __version__ = version_file.read().strip()
 
-def get_version():
+def version():
+    """Returns the current version of pyQuARC.
+    """
     return __version__
+
+def supported_formats():
+    """Returns a list of metadata formats supported by pyQuARC.
+       One of the formats in the list can be passed to ARC.validate()
+       Default is `echo-c`.
+
+       "dif10": Dif10 collectioin metadata format
+       "echo-c": Echo10 collection metadata format
```

Comment:
Nitpick: why "dif10" but not "echo10-c"?

Reply:
I assume these are formats followed by CMR.

Reply:
`umm-c` and `umm-g` are actually standards in CMR, but ECHO formats are just called ECHO collection/granule.

there's `dif` and `dif10` both, but only `echo10`.

so i guess, i was trying to make it the standard way of specifying collection or granule metadata by using `echo-c` and `echo-g` taking inspiration from `umm-c` and `umm-g`. Plus, `echo10-c` is just longer and harder to write than `echo-c`